knowledge/responses/its_my_personal_choice_to_eat_animals.md

```diff
@@ -1,7 +1,11 @@
-“Respect my opinion!”, “It's my personal choice to eat meat, don't force your views on to me!” - of all the excuses used to defend eating animal products, this could well be the most common. So is eating animals a personal choice or is it a little more complex than that? This is quite an interesting excuse as arguably it is a choice, one does actively choose to eat dead flesh and animal secretions, the same way that a domestic abuser actively chooses to abuse their partner or a rapist actively chooses to commit rape. 
+So is eating animals a personal choice or is it a little more complex than that? 
 
-Using this logic it would then be morally justifiable to beat a dog or kick a cat, as it is a personal choice to do so. In this situation ask the person, “if someone makes the personal choice to abuse a dog, does that make it morally justifiable?”. The problem is when non-vegans use this argument they have either become so detached from the fact that their animal products came from a living being, or they hold animals with such little regard that they don't consider the lives of their food worthy of contemplation because they think the consumption of animal products only affects them as individuals. 
+All acts come from personal choices. Some are inconsequential, like which socks to wear. Others have direct impacts on those who will suffer from that choice. It is your choice to eat animals. But by this logic it would then be morally justifiable to beat a dog or kick a cat, as it is a personal choice to do so. 
```

Comment:
```suggestion
All acts come from personal choices. Some are inconsequential, whereas others have direct impact on those who will suffer from that choice. It is your choice to eat animals. But by this logic it would then be morally justifiable to beat a dog or kick a cat, as it is a personal choice to do so. 
```